docs/reference/ccr/index.asciidoc

```diff
@@ -1,7 +1,7 @@
 [role="xpack"]
 [testenv="platinum"]
 [[xpack-ccr]]
-= Cross-cluster replication
+= Replicating data across multiple clusters
```

Comment:
I'm curious what is the reason for this change?

Reply:
It's not a necessity, but it seemed to align better with the titles of similar sections in the table of contents ("securing", "alerting", "monitoring").

Reply:
I think I prefer "Cross-cluster replication". It's easier to discover (`grep`, searches, etc.).

Reply:
Ok!  I've reverted that change.